sql/moz-fx-data-shared-prod/accounts_db_external/fxa_recovery_phones_v1/metadata.yaml

```diff
@@ -0,0 +1,18 @@
+---
+friendly_name: recoveryPhones table from production fxa database
+description: >
+  A mirror of the `recoveryPhones` table from the production `fxa` CloudSQL database,
+  excluding columns containing confidential data, updated daily to match the current state of the table.
+
+  See https://mozilla.github.io/ecosystem-platform/reference/database-structure#database-fxa
+owners:
+  - wclouser@mozilla.com
+labels:
+  application: accounts_backend
+  schedule: daily
+scheduling:
+  dag_name: bqetl_accounts_backend_external
+  # destination is the whole table, not a single partition,
+  # so don't use date_partition_parameter
+  date_partition_parameter: null
+
```

Comment:
CI appears to be complaining about the blank lines at the end of all the `metadata.yaml` and `query.sql` files.

Reply:
I checked and have a single blank line at the end of each file.  Hopefully that fixes it...

Reply:
Nope, the extra blank lines will have to be removed to make the CI happy.
```suggestion
```

Reply:
I feel like I had to have blanks the last time to make it happy.  Anyway, alright, force-pushed a patch that deleted all the new lines at the end

Reply:
That won't quite work either unfortunately.  It wants exactly one trailing newline character at the end of files, as I alluded to here:
> (though note the final non-blank line in each file should still end with a newline)

Reply:
I thought that's what I had before.  Ok, added a blank newline to each file.

Reply:
That still wasn't it (back to having 2 newline characters at the end of each file).

I committed the code suggestions I made in the PR to remove the extra blank lines and I now the CI is happy, so if you pull from your remote branch you can see how the CI wanted the files to be.